config/webpack.dev.config.js

```diff
@@ -2,13 +2,12 @@
 // time at the expense of creating larger, unoptimized bundles.
 const Merge = require('webpack-merge');
 const path = require('path');
-const webpack = require('webpack');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
 
 const commonConfig = require('./webpack.common.config.js');
 
 module.exports = Merge.smart(commonConfig, {
-  devtool: 'cheap-module-eval-source-map',
+  mode: 'development',
```

Comment:
The `development` mode sets the `devtool` to "eval". If the source maps are too hard to read then we can change this. "eval" is just the fastest to generate.